urbanopt/urbanopt-cli
# Disco commands

### Pull Request Description

- Adds cli calls for disco

To test: 

- create disco project: uo create --disco -p `project`
This should create a disco folder that contains cost_database.xlsx, technical_catalogue.json, config.json.
This should include the electrical network.json feature file

- run disco project
- post process disco project
- run opendss
- run disco: uo disco -s `scenario file`  -f `feature file`
This should create a `disco` folder in the scenario folder in the run directory.
The folder should include `run_upgrade_cost_analysis.log` and 
- post process disco : uo process --disco -s `scenario file` -f `feature file`
This should create a run_upgrade_cost_analysis.log file, output_summary.json file, and a job_outputs folder

TODO: 
- update technical_catalog.json after error is fixed
- add test for disco post processing
- update scenario and reporting gem branches once released

### Checklist (Delete lines that don't apply)

- [x] All ci tests pass (green)
- [x] An [issue](https://github.com/urbanopt/urbanopt-cli/issues) has been created (which will be used for the changelog)
- [x] This branch is up-to-date with develop
